cmd/terramate/cli/cli.go

```diff
@@ -859,15 +859,15 @@ func debugFiles(files []string, msg string) {
 	}
 }
 
-func (c *cli) gitFileSafeguards(checks stack.RepoChecks, shouldAbort bool) {
+func (c *cli) gitFileSafeguards(shouldAbort bool) {
```

Comment:
Nitpick: plain boolean arguments are never really super great. In this case it seems inconsequential (there's two call-sites) so there's no real urgent need to change it. Constants would be nice (`PanicOnError = true` and `WarnOnError=false` or something), but again, it's a nitpick.

Reply:
hm yes, I think it makes sense! I will try to improve this while doing the other PRs.